SR2021/2021-04-27-finals-reminder.md

```diff
@@ -0,0 +1,50 @@
+---
+to: All SR2021 teams
+subject: Student Robotics Finals
+---
+
+Hi,
+
+Now that the league stage is over, we wanted to give you some quick reminders
+about the schedule for next weekend's knockouts. We'd also love to hear about
+about your experiences competing in SR2021.
+
+## Feedback & Demographics Survey
+
+To help us understand how we're doing we'd really like to understand the make up
+of your team and their experiences with the competition this year. If they
+haven't done so already please encourage your competitors to complete our
```

Comment:
Given this is aimed at adults, perhaps adding something explicit like it helps with funding and understanding how we can best help competitors? Then team leaders are more likely to want to get competitors to complete it.

Reply:
Good idea. Could you suggest some wording to that effect?

Reply:
https://github.com/srobo/team-emails/pull/83/files#r620966282